src/bot_commands.c

```diff
@@ -2108,6 +2133,94 @@ static void FrogbotsDisable(void)
 	}
 }
 
+static void FrogbotsSetHealth(void)
```

Comment:
These functions introduce mode agnostic bot customizations unrelated to ToT. Would be nice to have their implementations and usages in a separate commit. This could even be a separate PR that's merged swiftly as they're isolated and clean.